### PR TITLE
User-defined build jobs

### PR DESCRIPTION
**Pull request by humitos**
Post announcing the new config key `build.jobs`.

Rendered version 🤖 : https://readthedocs-blog--162.org.readthedocs.build/user-defined-build-jobs/

Related to #160

### CODE REVIEW

**Comment by ericholscher** [comment]
Seemed easier to push up some edits instead of leaving ~15 comments. Feel free to adjust them as you see fit, but I tried to simplify the language.

**Review by humitos — COMMENTED** [review]
Thanks! I can't approve my own PR but this looks good to me.

I was asking myself why the post didn't appear on the front page... Now, I see we are in May not in April 🙃

**Review by agjohnson — APPROVED** [review]
This sounds good. I do still fell like calling the group of "jobs" a "step" is easier to relate to, but the terms user-defined jobs and predefined jobs do sound correct for the individual jobs at least.

This narrative content is some of what I think we should have had in our documentation. We could do more to outline the problem this solves, to help describe to the user why they are correct about needing custom build jobs.

File: user-defined-build-jobs.rst [hunk 1]
```diff
@@ -0,0 +1,83 @@
...
+
+
+The future of the builders
+--------------------------
+
+We are already discussing how we can expand this feature in the future.
+We'd like to support more complex build processes,
+or even projects that are not using Sphinx or MkDocs.
+That would open the door to a lot new possibilities.
+So, subscribe to our mailing list and stay tuned!
```
Comment (agjohnson) [inline user-defined-build-jobs.rst:73]: Cool, it lost my review note too.

Probably want a link to the subscribe page here.

File: user-defined-build-jobs.rst [hunk 2]
```diff
@@ -0,0 +1,83 @@
+.. post:: May 5, 2022
```
Comment (humitos) [inline user-defined-build-jobs.rst:1]:
```suggestion
.. post:: May 9, 2022
```

**Pull request merged**